docs/kb/error-codes-messages.md

```diff
@@ -1615,6 +1627,42 @@ requests.
 
 **Solution**: Resubmit the request using a valid [authentication](/#authentication) method.
 
+## [](#error-code-128)Error Code: 128
+
+```
+<rsp stat="fail" version="1.0">
+    <err code="128">Cannot create campaigns</err>
+</rsp>
+```
+
+**Problem**: Campaigns cannot be created for an account that has [Connected Campaigns](https://www.pardot.com/training/connected-campaigns-faq/) enabled.
+
+**Solution**: Create a campaign in [Salesforce](https://developer.salesforce.com/docs/api-explorer/sobject/Campaign).
+
+## [](#error-code-129)Error Code: 129
+
+```
+<rsp stat="fail" version="1.0">
+    <err code="129">Cannot modify campaign name or cost</err>
+</rsp>
+```
+
+**Problem**: Campaign name and cost cannot be modified via the API.
+
+**Solution**: Remove the name and cost fields from the request.
+
+## [](#error-code-130)Error Code: 130
+
+```
+<rsp stat="fail" version="1.0">
+    <err code="130">One or more of the specified fields are invalid</err>
+</rsp>
+```
+
+**Problem**: One or more of the fields in the CSV is not recognized.
```

Comment:
What does "not recognized" mean here? Is it functionally different than "invalid"? Consider using the same term as the error message.

Reply:
I guess I could clarify it further:
`One or more of the fields in the CSV is not a recognized standard or custom field`